### PR TITLE
Linked the rest of the skin widgets to the menu bar

### PR DESCRIPTION
**Pull request by jercaianu**
Related to #526 
Connected the rest of the skin widgets to the menu bar.

### CODE REVIEW

File: src/mixxx.cpp [hunk 1]
```diff
@@ -116,6 +116,14 @@ MixxxMainWindow::MixxxMainWindow(QApplication* pApp, const CmdlineArgs& args)
     m_pShoutcastManager = NULL;
 #endif
 
+    //Reset pointers to cos
```
Comment (daschuer) [inline src/mixxx.cpp]: Please use C++ initializations list. Feel free to clean up some other initializations as well.

File: src/mixxx.cpp [hunk 2]
```diff
@@ -116,6 +116,14 @@ MixxxMainWindow::MixxxMainWindow(QApplication* pApp, const CmdlineArgs& args)
...
+    m_pShowVinylControl = NULL;
```
Comment (daschuer) [inline src/mixxx.cpp]: do we need a #ifdef __VINYLCONTROL__ here?

File: src/mixxx.cpp [hunk 3]
```diff
@@ -843,11 +851,40 @@ void setVisibilityOptionState(QAction* pAction, ConfigKey key) {
     pAction->setChecked(pVisibilityControl != NULL ? pVisibilityControl->get() > 0.0 : false);
 }
 
-void MixxxMainWindow::toggleCheckedSamplers() {
-    m_pViewShowSamplers->blockSignals(true);
+void MixxxMainWindow::toggleCheckedMenuAction(QAction* menuAction, ConfigKey key) {
```
Comment (daschuer) [inline src/mixxx.cpp]: I do associate the term toogle with inverting the value. IMHO updateCheckedMenuAction fits better.

**Comment by daschuer** [comment]
Thank you for continuing to work on this.

**Comment by jercaianu** [comment]
I also added the ifdef there, however i don't see any harm being done by setting the pointer to NULL.

**Comment by daschuer** [comment]
Thank you very much! LGTM

**Pull request merged**